src/main.js

```diff
@@ -81,6 +90,31 @@ function showHomePage() {
   randomCoverBtn.classList.remove("hidden");
 }
 
+function saveCreatedCover(event) {
+  event.preventDefault();
+  var coverUrl = coverImageInput.value;
+  var title = titleInput.value;
+  var descriptor1 = descriptor1Input.value;
+  var descriptor2 = descriptor2Input.value;
```

Comment:
We'll look at refactoring these, in experimenting with declaring these variables globally the function didn't recognize them.